Src/IronPython/Runtime/Operations/PythonOps.cs

```diff
@@ -1954,7 +1954,24 @@ public static partial class PythonOps {
                 noLineFeed = false;
             }
 
-            string strCode = code as string;
+            string strCode;
```

Comment:
>  I would probably do the conversion higher up (at the `exec` level instead of here).

Sure.

Reply:
Got the conversion at the `exec` level, but `exec` can accept a whole range of types: string, code, file, Stream... They all get sorted out using `is`/`as`/`if`, down there in PythonOps. Did you mean to move all type matching to the level of `exec`, using overloads (would make things consistent with `eval` and `compile`) or just `bytes`?

Reply:
I meant moving the bytes to string conversions at the exec level.

At some point we could potentially move the whole thing (`QualifiedExec` and `UnqualifiedExec`) to `Builtin.cs` instead of having them in `PythonOps` since I think the the only reason they are in `PythonOps` because in 2.x exec was a statement. Probably not worth the effort...